### PR TITLE
Create min_stack.py

### PR DESCRIPTION
**Pull request by mavenzer**
### **Describe your change:**

Creating a Min Stack that can give minimum element in stack in constant time.

* [ ✔] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ✔] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [✔ ] This pull request is all my own work -- I have not plagiarized.
* [ ✔] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Why?  There is no description when or why this algorithm would be useful.  No doctests to prove that it works as intended.  Python method names should be in snake_case as discussed in CONTRIBUTING.md. Seems like a lot of code for an algorithm that could be implemented in 5 lines of Python.

**Comment by Abdujabbor** [comment]
add some doctests

**Review by mavenzer — COMMENTED** [review]
Updated min_stack.py with doc test and snake_case function definition.

File: data_structures/stacks/min_stack.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
 class MinStack:
+    """ Designing a Stack that can return Minimum element in Constant time"""
+
     def __init__(self):
         """ Creating stack minStack and size Variable"""
         self.stack = []
         self.minStack = []
         self.size = 0
```
Comment (cclauss) [inline data_structures/stacks/min_stack.py:8]: Please see https://github.com/TheAlgorithms/Python/pull/2477#discussion_r494864246 and implement [`.__iter__()`](https://docs.python.org/3/reference/datamodel.html#object.__iter__) and `.__len__()` methods for this class.  `.__iter__()` should return values in min-to-max order.

**Pull request closed without merging**